Model/CustomItemImportModel.php

```diff
@@ -94,22 +96,32 @@ public function import(Import $import, array $rowData, CustomObject $customObjec
             $merged = true;
         }
 
-        $this->linkContacts($customItem, $contactIds);
+        $this->linkContacts($customItem, $contactIds, $importLogDto);
 
         return $merged;
     }
 
     /**
      * @param int[] $contactIds
      */
-    private function linkContacts(CustomItem $customItem, array $contactIds): CustomItem
+    private function linkContacts(CustomItem $customItem, array $contactIds, ?ImportLogDTO $importLogDto): void
     {
         foreach ($contactIds as $contactId) {
+            if (!$this->entityManager->getRepository(Lead::class)->exists((string) $contactId)) {
```

Comment:
```suggestion
            $leadRepository = $this->entityManager->getRepository(Lead::class);
            if (method_exists($leadRepository,'exists') && !$leadRepository->exists((string) $contactId)) {
```

Reply:
please add method_exist otherwise it will fail on community untill mc-cs PR does not get merged.